cmd/rpcdaemon/graphql/graph/helpers.go

```diff
@@ -67,15 +68,12 @@ func convertDataToIntP(abstractMap map[string]interface{}, field string) *int {
 
 	switch v := abstractMap[field].(type) {
 	case hexutil2.Uint64:
+	case *hexutil2.Uint:
```

Comment:
we should use a `,` in such cases, i.e. `case hexutil2.Uint64, hexutil2.Uint`

Reply:
Can't use comma in type switch, so I handled them separeately